### PR TITLE
PrestaShop Feature Freeze: Important Features of Version 8.0

### PR DESCRIPTION
**Pull request by coffrep**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Adding an article about the version 8.0 feature freeze, to be posted when feature freeze is done. Could someone review the technical part of the article, especially the "develop" term in the post. Thanks!
| Fixed ticket? | None

### CODE REVIEW

File: news/_posts/2022-06-06-prestashop-feature-freeze-v8.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ - 8.0
+---
+
+Dear developers and contributors,
+
+We are very pleased to be announcing the feature freeze of PrestaShop version 8.0. In this post, we will be looking at the most important features of this feature freeze, letting you know why we have frozen new features and explain what this means for the next few months.
+
+### Feature freeze is now effective
+
+Therefore, as of today, the feature freeze is now effective and no new features will be added to version 8.0, so we can work on implementing new features and working on their quality.
```
Comment (matks) [inline news/_posts/2022-06-06-prestashop-feature-freeze-v8.md]: I'm a little confused by the sequence
>  no new features will be added to version 8.0
then
> we can work on implementing new features 

I suggest
```suggestion
Therefore, as of today, the feature freeze is now effective and no new features will be added to version 8.0, so we can work on polishing the 8.0 features and working on their quality.
```

File: news/_posts/2022-06-06-prestashop-feature-freeze-v8.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+In the next few months, we will still be welcoming new features. We will be creating the 8.0.x branch to stabilize the code of the next 8.0 and new features will be added on “develop”.
+
+### Most important features of version 8.0
+
+In this section, we wanted to highlight the most important features of this version 8.0 feature freeze, visible below:
+1. Update to[ Symfony 4.4](https://symfony.com/releases/4.4) (LTS version, support until November 23)
+2. Compatibility with[ PHP 8](https://www.php.net/) / 8.1
+3. Many different items regarding the open source project separation, removing all dependencies to the company to make the project as neutral as possible
```
Comment (matks) [inline news/_posts/2022-06-06-prestashop-feature-freeze-v8.md]: Maybe we could link to a more detailed explanation. The demo done by @ttoine maybe?

**Review by matks — COMMENTED** [review]
@coffrep Nice I see you were able to fix the first issue and now this looks a lot better.

The format is valid, we could publish this now and it would work. I reviewed and noticed some things you might want to improve, just tell me.

Also I guess we feature freeze to be done before publishing this article?

File: news/_posts/2022-06-06-prestashop-feature-freeze-v8.md (hunk 2 above)
Comment (coffrep) [reply news/_posts/2022-06-06-prestashop-feature-freeze-v8.md]: Hi Matks, would this video be the right one? https://youtu.be/LyNUS0STl4Q?t=943 Or this one, but it's more 2020-2021 https://youtu.be/Bp8ef2jjIBw Let me know.

Comment (coffrep) [reply news/_posts/2022-06-06-prestashop-feature-freeze-v8.md]: And yes let's wait for the Feature Freeze to be effective before posting.

Comment (matks) [reply news/_posts/2022-06-06-prestashop-feature-freeze-v8.md]: I was thinking about this one https://www.youtube.com/watch?t=128&v=jwOy3gx-MNs&feature=youtu.be which is in English 🇬🇧

Comment (matks) [reply news/_posts/2022-06-06-prestashop-feature-freeze-v8.md]: It was presented in October 2021
https://build.prestashop.com/news/public-oss-demo-10-2021/

File: news/_posts/2022-06-06-prestashop-feature-freeze-v8.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Therefore, as of today, the feature freeze is now effective and no new features will be added to version 8.0, so we can work on polishing the 8.0 features and working on their quality.
+
+In the next few months, we will still be welcoming new features. We will be creating the 8.0.x branch to stabilize the code of the next 8.0 and new features will be added on “develop”.
+
+### Most important features of version 8.0
+
+In this section, we wanted to highlight the most important features of this version 8.0 feature freeze, visible below:
+1. Update to[ Symfony 4.4](https://symfony.com/releases/4.4) (LTS version, support until November 23)
+2. Compatibility with[ PHP 8](https://www.php.net/) / 8.1
```
Comment (atomiix) [inline news/_posts/2022-06-06-prestashop-feature-freeze-v8.md]: I think we can just say PHP 8.1 as it implicitly means that it is compatible with PHP 8.0 too?
```suggestion
1. Update to [Symfony 4.4](https://symfony.com/releases/4.4) (LTS version, support until November 23)
2. Compatibility with [PHP 8.1](https://www.php.net/)
```

File: news/_posts/2022-06-06-prestashop-feature-freeze-v8.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+### Feature freeze is now effective
+
+Therefore, as of today, the feature freeze is now effective and no new features will be added to version 8.0, so we can work on polishing the 8.0 features and working on their quality.
+
+In the next few months, we will still be welcoming new features. We will be creating the 8.0.x branch to stabilize the code of the next 8.0 and new features will be added on “develop”.
+
+### Most important features of version 8.0
+
+In this section, we wanted to highlight the most important features of this version 8.0 feature freeze, visible below:
```
Comment (MatShir) [inline news/_posts/2022-06-06-prestashop-feature-freeze-v8.md:26]:
```suggestion
In this section, we wanted to highlight the most important features of this version 8.0 feature freeze, visible below:
1. Update to [Symfony 4.4](https://symfony.com/releases/4.4) (LTS version, support until November 23)
2. Compatibility with [PHP 8](https://www.php.net/) / 8.1
3. Many different items regarding the open source project separation, removing all dependencies to the company to make the project as neutral as possible
4. A redesign of the product page:
- Bulk edition enhanced for the product combinations
- Reorganization of all the input information
- Better image management (replacing images, bulk edition)
- Performances improvement (categories and combinations)
5. And new password policy making PrestaShop more secure and safe

Thank you to all the contributors that have helped with PrestaShop version 8.0: the community, our users and developers are what made this version possible. Looking forward to the final release of version 8.0! ```

File: news/_posts/2022-06-06-prestashop-feature-freeze-v8.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+1. Update to [Symfony 4.4](https://symfony.com/releases/4.4) (LTS version, support until November 23)
+2. Compatibility with [PHP 8.1](https://www.php.net/)
+3. Many different items regarding the open source project separation, removing all dependencies to the company to make the project as neutral as possible
+4. A redesign of the product page:
+- Bulk edition enhanced for the product combinations
+- Reorganization of all the input information
+- Better image management (replacing images, bulk edition)
+- Performances improvement (categories and combinations)
+
+Thank you to all the contributors that have helped with PrestaShop version 8.0: the community, our users and developers are what made this version possible. Looking forward to the final release of version 8.0!
```
Comment (MatShir) [inline news/_posts/2022-06-06-prestashop-feature-freeze-v8.md:37]:
```suggestion
5. And a new password policy making PrestaShop more secure and safe.

Thank you to all the contributors that have helped with PrestaShop version 8.0: the community, our users and developers are what made this version possible. Looking forward to the final release of version 8.0! 
```

File: news/_posts/2022-06-06-prestashop-feature-freeze-v8.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+In the next few months, we will still be welcoming new features. We will be creating the 8.0.x branch to stabilize the code of the next 8.0 and new features will be added on “develop”.
+
+### Most important features of version 8.0
+
+In this section, we wanted to highlight the most important features of this version 8.0 feature freeze, visible below:
+1. Update to [Symfony 4.4](https://symfony.com/releases/4.4) (LTS version, support until November 23)
+2. Compatibility with [PHP 8.1](https://www.php.net/)
+3. Many different items regarding the open source project separation, removing all dependencies to the company to make the project as neutral as possible
+4. A redesign of the product page:
```
Comment (MatShir) [inline news/_posts/2022-06-06-prestashop-feature-freeze-v8.md]:
```suggestion
4. An experimental feature, the redesign of the product page:
```

File: news/_posts/2022-06-06-prestashop-feature-freeze-v8.md (hunk 2 above)
Comment (matks) [reply news/_posts/2022-06-06-prestashop-feature-freeze-v8.md]:
```suggestion
3. Many different items regarding the [open source project separation](https://www.youtube.com/watch?t=128&v=jwOy3gx-MNs&feature=youtu.be), removing all dependencies to the company to make the project as neutral as possible
```

File: news/_posts/2022-06-06-prestashop-feature-freeze-v8.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title:  "PrestaShop Version 8.0 Feature Freeze: Important Features"
+subtitle: "Feature Freeze: an Inside Look at Most Important Features"
+date:   2022-06-06 09:10:11
```
Comment (matks) [inline news/_posts/2022-06-06-prestashop-feature-freeze-v8.md]:
```suggestion
date:   2022-06-09 12:00:00
```

**Review by matks — COMMENTED** [review]
Good for me
- I update day and hour for publication
- I suggest to add link to Public OSS demo about open source separation

**Review by MatShir — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Published ! @coffrep

**Pull request merged**